src/main/java/com/project/momo/dto/member/MemberInfoResponse.java

```diff
@@ -0,0 +1,30 @@
+package com.project.momo.dto.member;
+
+import com.project.momo.entity.Member;
+import com.project.momo.entity.Payment;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Getter
+@NoArgsConstructor
```

Comment:
```suggestion
@NoArgsConstructor(AccessLevel.PRIVATE)
```
사용하지 않는다면 private으로 막아두는 것이 좋지 않을까요?

Reply:
접근 제어에 신경쓰겠습니다ㅜㅜ